src/common/workflowRun.ts

```diff
@@ -0,0 +1,53 @@
+import { getOctokit } from '@actions/github';
+
+const HTTP_OK = 200;
+
+export enum WorkflowRunConclusion {
+  ActionRequired = 'action_required',
+  Cancelled = 'cancelled',
+  Failure = 'failure',
+  Neutral = 'neutral',
+  Success = 'success',
+  Skipped = 'skipped',
+  Stale = 'stale',
+  TimedOut = 'timed_out',
+}
+
+export const getLastWorkflowRunConclusion = async (
+  octokit: ReturnType<typeof getOctokit>,
```

Comment:
Can we use OctoKit Types instead?

https://www.npmjs.com/package/@octokit/types

Reply:
Will check. The PR is still a work in progress, I opened the PR to check how the action is working now :)